src/renderer/component/publishForm/view.jsx

```diff
@@ -442,6 +466,38 @@ class PublishForm extends React.PureComponent {
     this.setState({ id });
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.uploadStatus === "error") {
+      alert("UPLOAD FAILED, PLEASE TRY AGAIN.");
```

Comment:
we should display this in a better way (although, makes sense to have that in there for testing/dev purposes)

Reply:
`doAlertError` exists as an action, and can be an appropriate way of handling generic errors